### PR TITLE
Add falcon-7b-instruct tutorial and ModelServer -> Server

### PR DESCRIPTION
**Pull request by samos123**
Changes:
* Change the ModelServer to Server
* Update API docs from latest main of substratus
* Add tutorial for end-to-end loading and fine tuning falcon-7b-instruct model
* Change quickstart to use falcon-7b-instruct for more impressive base model
* Change notebook conversion script to also render output of the cells

### CODE REVIEW

**Review by brandonjbjelland — APPROVED** [review]
👩‍🎤

File: docs/quickstart.md [hunk 1]
```diff
@@ -57,67 +57,90 @@ docker run -it \
...
+          [0m[34;1m"logprobs"[0m[1;39m: [0m[1;30mnull[0m[1;39m,
+          [0m[34;1m"finish_reason"[0m[1;39m: [0m[0;32m"length"[0m[1;39m
+        [1;39m}[0m[1;39m
+      [1;39m][0m[1;39m,
+      [0m[34;1m"usage"[0m[1;39m: [0m[1;39m{
+        [0m[34;1m"prompt_tokens"[0m[1;39m: [0m[0;39m11[0m[1;39m,
+        [0m[34;1m"completion_tokens"[0m[1;39m: [0m[0;39m12[0m[1;39m,
+        [0m[34;1m"total_tokens"[0m[1;39m: [0m[0;39m23[0m[1;39m
+      [1;39m}[0m[1;39m
+    [1;39m}[0m
```
Comment (brandonjbjelland) [inline docs/quickstart.md:131]: Do these color characters work on the actual site? Github doesn't render them properly.

Comment (samos123) [reply docs/quickstart.md:131]: Nope it doesn't render them :(
image

Comment (samos123) [reply docs/quickstart.md:131]: it looked so nice in my vscode

Comment (brandonjbjelland) [reply docs/quickstart.md:131]: why can't everything look as nice as vscode? 😭

**Pull request merged**